compile-build

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-set -e
+set -e # Exit immediately if something fails
+export NODE_ENV=production
+export RAILS_ENV=development
+echo 'Updating yarn'
+bundle exec yarn
```

Comment:
Why does this need `bundle exec`?  I don't think `yarn` is a Bundler dependency.

Reply:
Yarn is needed to build the node dependencies. So bundle exec yarn will do this.